reports/2023-02-01-daskhub-induced-evictions.md

```diff
@@ -0,0 +1,20 @@
+# 2023-02-01 Heavy use of dask-gateway induced critical pod evictions
+
+## Timeline
+
+All times in UTC+1
+
+- 2023-02-01 - [Summary of issue updated between ~8-9 PM](https://github.com/2i2c-org/infrastructure/issues/2126#issuecomment-1412554908)
+
+## What went wrong
+
+- I believe various critical pods on core nodes pods got evicted when prometheus started scraping from ~200 nodes metrics exporters
```

Comment:
Can we do better than belief? Is there not evidence to support your hypothesis here? That's what I'd expect the incident report/postmortem to reveal through analysis of the contributing factors/root causes/triggers.